Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/spinning-loader/SpinningLoader.tsx

```diff
@@ -0,0 +1,13 @@
+﻿import React from 'react';
+import './SpinningLoader.scss';
+
+const SpinningLoader = () => {
+    return(<div className="lds-ring">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
```

Comment:
What is the purpose of so many divs?

Reply:
styling the loader, take a look at  SpinningLoader.scss